grpc.go

```diff
@@ -394,29 +394,66 @@ func (a *agentGRPC) postExecProcess(ctr *container, proc *process) error {
 // path set by the spec, since we will always ignore it. Indeed, it makes no
 // sense to rely on the namespace path provided by the host since namespaces
 // are different inside the guest.
-func (a *agentGRPC) updateContainerConfigNamespaces(config *configs.Config) error {
+func (a *agentGRPC) updateContainerConfigNamespaces(config *configs.Config, ctr *container) {
+	pidNs := false
+	utcNs := false
+	ipcNs := false
```

Comment:
can it be 
`var pidNs, utcNs, ipcNs = false, false, false `
to make it simple

Reply:
Or simply var foo, bar bool. The default value for bool is false.